governance/polls/Update the wstETH oracle to assume a 1:1 stETH:ETH peg - January 8, 2024.md

```diff
@@ -0,0 +1,50 @@
+---
+title: Update the wstETH oracle to assume a 1:1 stETH:ETH peg - January 08, 2024
```

Comment:
```suggestion
title: SparkLend Ethereum - Update the wstETH oracle to assume a 1 to 1 stETH to ETH peg - January 08, 2024
```

":" character is banned, please rename the file. See the poll checklists https://forum.makerdao.com/t/govalpha-handover-documentation-index/21365

Reply:
Ah, yes. I did remember that at some point, but forgot to make the change!